src/Microsoft.AspNetCore.SystemWebAdapters/FeatureCollectionExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.AspNetCore.Http.Features;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters;
+
+internal static class FeatureCollectionExtensions
+{
+    internal static TFeature GetRequired<TFeature>(this IFeatureCollection features)
```

Comment:
Is GetRequiredFeature not available?
https://github.com/dotnet/aspnetcore/blob/64f95ef2377403e075cced875363419755d317a4/src/Extensions/Features/src/FeatureCollectionExtensions.cs#L21

Reply:
We're targeting .NET 6 where it is not. I've looked into switching to that on .NET 7+, but we have internals visible to and it ends up causing compiler warnings of multiple resolved types.